src/java/org/apache/cassandra/index/sai/SSTableIndex.java

```diff
@@ -375,6 +379,10 @@ public int hashCode()
 
     public List<CloseableIterator<PrimaryKeyWithSortKey>> orderResultsBy(QueryContext context, List<PrimaryKey> keys, Orderer orderer, int limit, long totalRows) throws IOException
     {
+        context.checkpoint();
+        context.addSstablesHit(1);
```

Comment:
This doesn't seem hit by any query in `SlowSAIQueryLoggerTest`, the test passes without this change.